crates/subspace-node/src/domain/evm_chain_spec.rs

```diff
@@ -164,67 +168,88 @@ pub enum SpecId {
     Local,
 }
 
-pub fn get_testnet_genesis_by_spec_id(spec_id: SpecId) -> GenesisConfig {
+pub struct GenesisDomainParams {
+    pub operator_signing_key: OperatorPublicKey,
+}
+
+pub fn get_testnet_genesis_by_spec_id(spec_id: SpecId) -> (GenesisConfig, GenesisDomainParams) {
     match spec_id {
         SpecId::Dev => {
             let accounts = get_dev_accounts();
-            testnet_genesis(
-                accounts.clone(),
-                // Alith is Sudo
-                Some(accounts[0]),
-                vec![(
-                    accounts[0],
-                    AccountId32ToAccountId20Converter::convert(
-                        get_from_seed::<sr25519::Public>("Alice").into(),
-                    ),
-                )],
-                1000,
+            (
+                testnet_genesis(
+                    accounts.clone(),
+                    // Alith is Sudo
+                    Some(accounts[0]),
+                    vec![(
+                        accounts[0],
+                        AccountId32ToAccountId20Converter::convert(
+                            get_from_seed::<sr25519::Public>("Alice").into(),
+                        ),
+                    )],
+                    1000,
+                ),
+                GenesisDomainParams {
+                    operator_signing_key: get_public_key_from_seed::<OperatorPublicKey>("Alice"),
+                },
```

Comment:
We can just add `operator_signing_key` as an argument of `testnet_genesis` that would be more readable IMHO.

Reply:
Would prefer leaving it as since adding another field would be not lead to much noise.

Reply:
Non-blocker and don't have strong opinion but noise is introduced here I think.